pkg/templates/nstemplatetiers/nstemplatetier_generator_test.go

```diff
@@ -31,37 +31,108 @@ func TestCreateOrUpdateResources(t *testing.T) {
 	err := apis.AddToScheme(s)
 	require.NoError(t, err)
 	logf.SetLogger(zap.Logger(true))
+	mockCreate := func(clt *testsupport.FakeClient) func(ctx context.Context, obj runtime.Object, opts ...client.CreateOption) error {
```

Comment:
Do we still need the mocks `mockCreate` and `mockUpdate` - it looks like the latest fake client in common should cover the cases.

Reply:
removed in 529d555